interfaces/interfaces.c

```diff
@@ -29,35 +29,32 @@ static const char *entity_list_version = "VClientEntityList003";
 static const char *surface_version = "VGUI_Surface030";
 static const char *vgui_panel_version = "VGUI_Panel009";
 
+/// This is a "bad" way to get the dl paths. Since the dynamic library has all the context of the tf2 executable file,
+/// we can just do the paths with in relativity to the executable file, which is much more agnostic to the operating system.
+
```

Comment:
Nice, this is a great addition and good info. Removed comment in my latest push.